config/identify_system.go

```diff
@@ -6,16 +6,18 @@ import (
 )
 
 // Figure out if we're self-hosted or on RDS, as well as what ID we can use - Heroku is treated separately
-func identifySystem(config ServerConfig) (systemType string, systemScope string, systemID string) {
+func identifySystem(config ServerConfig) (systemType string, systemScope string, systemScopeFallback string, systemID string) {
 	// Allow overrides from config or env variables
 	systemType = config.SystemType
 	systemScope = config.SystemScope
+	systemScopeFallback = config.SystemScopeFallback
 	systemID = config.SystemID
 
 	if config.AwsDbInstanceID != "" || systemType == "amazon_rds" {
 		systemType = "amazon_rds"
 		if systemScope == "" {
-			systemScope = config.AwsRegion
+			systemScope = config.AwsAccountID + "." + config.AwsRegion
```

Comment:
Come to think of it, if `AwsAccountId` is unset (empty), we should probably avoid prepending the period, right? And in that case we don't need to set the fallback scope at all?

Reply:
Yeah, good point - I'll adjust this.